assets/src/scripts/monitoring-location/components/hydrograph/data-indicator.js

```diff
@@ -0,0 +1,46 @@
+import {select} from 'd3-selection';
```

Comment:
This was called data-loading-indicator. I change the name as it now includes the No Data Overlay message indicator.